packages/es-components/src/components/controls/textbox/InputBase.js

```diff
@@ -7,6 +7,7 @@ export default styled.input`
   box-sizing: border-box;
   color: ${props => props.theme.colors.black};
   font-size: ${props => props.theme.font.baseFontSize};
+  font-face: ${props => props.theme.font.baseFontFace};
```

Comment:
Of note: because `baseFontFase` is not defined on the Theme in older versions, when people do not have the updated Theme package, this interpolation will return `undefined` and the style rule will simply not get set (per [this comment](https://github.com/styled-components/styled-components/issues/402#issuecomment-737123943) on a closed Issue).

That should work fine, as the Textarea and Textbox components will just revert to the same font display behavior we have now.